sass/_base.scss

```diff
@@ -31,4 +44,26 @@ a {
     text-decoration: none;
     color: inherit;
     font-size: inherit;
+}
+
+.cta-SeeMore {
```

Comment:
This style doesn't look like a base style. Looks more like an atom. Move it to the atoms folder.